PIM/Integration/07_ManageVariantSets.md

```diff
@@ -0,0 +1,182 @@
+  The editing toolbar is displayed above the attribute list.
+
+  > [Info] Generally, a variant inherits all attribute values from its master product. These attribute values are locked in the variant and can only be edited in the master product. Attribute values that differ from the master product and must be maintained individually in the variant itself must be added to the list of changeable attributes.
+
+3. Click the [ADD AND GO BACK] button.   
+  The *Add changeable attributes* view is closed. The selected attributes are displayed in the *Changeable Attributes* box.
+
+  ![Changeable attributes added](../../Assets/Screenshots/PIM/Settings/VariantSets/ChangeableAttributesAdded.png "[Changeable attributes added]")
+
+4. Click the [SAVE] button in the upper right corner.   
+  The *Create variant set* view is closed. The new variant set has been saved and is displayed in the list of variant sets.
+
+
+
+## Edit a variant set
+
+After you have created a variant set, you can edit it. However, only some fields are editable. The assigned attribute set cannot be subsequently modified.
+
+#### Prerequisites
+
+At least one variant set has been created, see [Create a variant set](#create-a-variant-set).
+
+#### Procedure
+*PIM > Settings > Tab VARIANT SETS*
+
+![Variant sets](../../Assets/Screenshots/PIM/Settings/VariantSets/VariantSets.png "[Variant sets]")
+
+1. Click the variant set you want to edit in the list of variant sets.   
+  The *Edit variant set* view is displayed.
+
+  ![Edit variant set](../../Assets/Screenshots/PIM/Settings/VariantSets/EditVariantSet.png "[Edit variant set]")
+
+2. Edit the desired data of the variant set in the corresponding fields.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Edit variant set* view is closed.
+
+[comment]: <> (I have to refresh the list to apply the changes - is that a bug?)
+
+
+
+## Remove an attribute from the variant set
+
+You can remove defining and changeable attributes from the variant set. Note, that at least one defining attribute is assigned. If one or more changeable attributes has been assigned to the variant set, also at least one changeable attribute must be assigned.
+
+[comment]: <> (is that a bug?)
+
+#### Prerequisites
+
+A variant set has been created, see [Create a variant set](#create-a-variant-set).
+
+#### Procedure
+*PIM > Settings > Tab VARIANT SETS*
+
+![Variant sets](../../Assets/Screenshots/PIM/Settings/VariantSets/VariantSets.png "[Variant sets]")
+
+1. Click the variant set you want to edit in the list of variant sets.   
+  The *Edit variant set* view is displayed.
+
+  ![Edit variant set](../../Assets/Screenshots/PIM/Settings/VariantSets/EditVariantSet.png "[Edit variant set]")
+
+2. In the *Defining attributes* box or *Changeable attributes* box, select the checkboxes of the attributes you want to remove.     
```

Comment:
```suggestion
2. In the *Defining attributes* box or the *Changeable attributes* box, select the checkboxes of the attributes you want to remove.     
```
Select the checkboxes of the attributes you want to remove (either) in the *Defining attributes* box or in the *Changeable attributes* box.